libmpv/qt_opengl/mpvwidget.cpp

```diff
@@ -75,7 +75,7 @@ void MpvWidget::initializeGL()
 
 void MpvWidget::paintGL()
 {
-    mpv_opengl_fbo mpfbo{static_cast<int>(defaultFramebufferObject()), width(), height(), 0};
+    mpv_opengl_fbo mpfbo{static_cast<int>(defaultFramebufferObject()), (int) (width() * devicePixelRatio()), (int) (height() * devicePixelRatio()), 0};
```

Comment:
Use `qreal dpr = devicePixelRatio()` once and refer to that variable instead of calling `devicePixelRatio` twice.

Use new-style casts such as `int(width()*dpr)`, not C-style casts.